test/integration/test-video-players-helper.js

```diff
@@ -153,7 +153,7 @@ export function runVideoPlayerIntegrationTests(createVideoElementFunc) {
     })
     .then(() => {
       const video = createVideoElementFunc(fixture);
-      video.setAttribute('autoplay', '');
+      video.setAttribute('autoplay', true);
```

Comment:
no boolean value for attribute ( disallowed by HTML5) this needs to remain video.setAttribute('autoplay', '');